en/docs/develop/identity-governance-rest-api.md

```diff
@@ -3,16 +3,164 @@ template: templates/swagger.html
+    | Max failed attempts   | sso.login.recaptcha.on.max.failed.attempts   | Number of failed attempts allows without showing the captcha |
+
+??? "Account Management Policies"
+
+    * Lock Idle Accounts
+    
+    | Property display name | Property name | Purpose |
+    | ------- | -------|---|
+    | Enable   | suspension.notification.enable   |  |
+    | Lock Account After (days)   | suspension.notification.account.disable.delay   |  |
+    | Alert Users before account locking in days (comma-separated list)   | suspension.notification.delays   |  |
+    
+    * Account Recovery
+    
+    | Property display name | Property name | Purpose |
+    | ------- | -------|---|
+    | Enable Notification Based Password Recovery   | Recovery.Notification.Password.Enable   |  |
+    | Enable reCaptcha for Password Recovery   | Recovery.ReCaptcha.Password.Enable   |  |
+    | Enable Security Question Based Password Recovery   | Recovery.Question.Password.Enable   |  |
+    | Number Of Questions Required For Password Recovery   | Recovery.Question.Password.MinAnswers   |  |
+    | Enable reCaptcha for Security Questions Based Password Recovery   | Recovery.Question.Password.ReCaptcha.Enable   | Show captcha for challenge question based password recovery |
+    | Max Failed Attempts for ReCaptcha   | Recovery.Question.Password.ReCaptcha.MaxFailedAttempts   |  |
+    | Enable Username Recovery   | Recovery.Notification.Username.Enable   |  |
+    | Enable reCaptcha for Username Recovery   |  Recovery.ReCaptcha.Username.Enable  |  |
+    | Enable Internal Notification Management   | Recovery.Notification.InternallyManage  | Set false if the client application handles notification sending |
+    | Notify when Recovery Success   | Recovery.NotifySuccess   |  |
+    | Notify when Questions Based Recovery Starts   | Recovery.Question.Password.NotifyStart   |  |
+    | Recovery Link Expiry Time   | Recovery.ExpiryTime   |  |
+    | Enable forced challenge questions   | Recovery.Question.Password.Forced.Enable   | Force users to provide answers to challenge questions during sign in |
+    | Recovery callback URL regex   | Recovery.CallbackRegex   |  |
+    
+    * User Self Registration
+    
+    | Property display name | Property name | Purpose |
+    | ------- | -------|---|
+    | Enable Self User Registration  | SelfRegistration.Enable  | Enable self user registration |
+    | Enable Account Lock On Creation | SelfRegistration.LockOnCreation  | Lock user account during user registration |
+    | Internal Notification Management  | SelfRegistration.Notification.InternallyManage  | Set false if the client application handles notification sending |
+    | Enable reCaptcha  | SelfRegistration.ReCaptcha  | Enable captcha verification during self registration |
+    | User self registration verification link expiry time  | SelfRegistration.VerificationCode.ExpiryTime  | Set the number of minutes the user self registration verification mail would be valid |
```

Comment:
Set the number of minutes for which the user self registration verification mail is valid